gmbuilder.Dockerfile

```
@@ -0,0 +1,9 @@
+FROM golang:1.20.3
+
+WORKDIR /
+
+ADD ./ /cosmos-sdk/
+
+COPY script.sh /script.sh
```

you can add the permission directly in the copy and skip/remove the `chmod` layer (line 8)

```suggestion
COPY --chmod=+x script.sh /script.sh
```
or:
```suggestion
COPY --chmod=0744 script.sh /script.sh
```

*please, try to build it locally first*